js/src/contract/item.ts

```diff
@@ -432,7 +434,7 @@ export class ItemProgram {
         tokenMint: accounts.itemClassMint,
         parentMint: accounts.parentMint,
         parentIndex: args.parentClassIndex,
-        parent: accounts.parentMint
+        parent: accounts.parentMint && !isNull(args.parentClassIndex)
```

Comment:
Why are we bringing in a new dependency to check for null? Based on what I see it being used for, a `!` or `!!` should suffice.

```
$ node
> !null
true
> !undefined
true
> !(void 0)
true
> !!null
false
> !!undefined
false
> !!(void 0)
false
```

Also curious what your intention is with adding `&& !isNull`? In the case the parentClassIndex is indeed `null`, this statement would return false, which seems counter to what the types for this function's arguments require, which would be `null` not false.

```
$ node
> true && !!null
false
```

Reply:
It's used as the conditional in a ternary, the return comes after:
https://github.com/raindrops-protocol/raindrops/blob/bb3807bf8088c875cf2ea1ca87ba79d406766f5c/js/src/contract/item.ts#L437-L441

Reply:
Oh, wow. It sure is. Haha. I had tunnel vision on that one.